st2api/st2api/config.py

```diff
@@ -28,10 +28,23 @@
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
+def parse_args(args=None):
+    CONF(args=args, version=VERSION_STRING)
+
+
+def register_opts():
+    _register_common_opts()
+    _register_app_opts()
+
+
 def _register_common_opts():
     common_config.register_opts()
 
 
+def get_logging_config_path(service='api'):
```

Comment:
The problem is that we don't have one `config.py` per service.

First I went with adding `get_logging_config_path` method to `config.py` module approach, but it broke down with the services in `st2actions/` module since those service doesn't have separate config (see https://github.com/StackStorm/st2/pull/1656/files#diff-32a22336e395c4f129c6b33e1c91545eR96 for details).

I would actually be OK with refactoring those services and also creating a config for resulttracker and notifier (move both in a package under st2action - st2actions/{notifier,result)tracker}/config.py. This approach is probably better anyway since it would make notifier and result tracker follow established and consistent patterns from other services.

WDYT?